extension/json.go

```diff
@@ -62,15 +69,27 @@ func (updateResponse *UpdateResponse) MarshalJSON() ([]byte, error) {
 			app.UpdateCheck.URLs.URLs = append(app.UpdateCheck.URLs.URLs, URL{
 				Codebase: url,
 			})
+
 			app.UpdateCheck.Manifest = &Manifest{
 				Version: extension.Version,
 			}
 
 			pkg := Package{
 				Name:     extensionName,
 				SHA256:   extension.SHA256,
+				FP:       extension.SHA256,
 				Required: true,
 			}
+
+			if ok {
```

Comment:
While I myself love the "comma, ok" idiom, and urge everyone to follow it, here is an example of a situation where it might be worth giving the `ok` variable a more accurate name. This is due to two reasons:
1. It's far removed from the place it's declared -> easy to forget what exactly is `ok`.
2. It is happening inside a block with `Status == "ok"` which makes this specific place even more confusing.

Consider renaming it to `pinfoFound` or something.